### PR TITLE
Bootstrap scheduler and remove a MetadataRepo call from iasql ops

### PR DESCRIPTION
**Pull request by depombo**

### CODE REVIEW

File: src/router/db.ts [hunk 1]
```diff
@@ -76,17 +78,29 @@ db.get('/remove/:dbAlias', async (req, res) => {
 
 db.post('/apply', async (req, res) => {
   const { dbAlias, dryRun } = req.body;
+  if (!dbAlias) return res.status(400).json(
+    `Required key(s) not provided: ${[
+      'dbAlias',
+    ].filter(k => !req.body.hasOwnProperty(k)).join(', ')}`
+  );
```
Comment (dfellis) [inline src/router/db.ts:85]: If there's only one key, do we need the array and filter stuff?

File: src/router/db.ts [hunk 2]
```diff
@@ -76,17 +78,29 @@ db.get('/remove/:dbAlias', async (req, res) => {
...
   }
 });
 
 db.post('/sync', async (req, res) => {
   const { dbAlias, dryRun } = req.body;
+  if (!dbAlias) return res.status(400).json(
+    `Required key(s) not provided: ${[
+      'dbAlias',
+    ].filter(k => !req.body.hasOwnProperty(k)).join(', ')}`
+  );
```
Comment (dfellis) [inline src/router/db.ts:100]: Same here

**Review by dfellis — APPROVED** [review]
Only minor comments

File: src/router/db.ts (hunk 1 above)
Comment (depombo) [reply src/router/db.ts:85]: that is true

File: src/router/db.ts (hunk 2 above)
Comment (dfellis) [reply src/router/db.ts:100]:
```suggestion
  if (!dbAlias) return res.status(400).json('Required key not provided: dbAlias');
```

**Pull request merged**